module/behaviour/strategy/SoccerStrategy/src/SoccerStrategy.cpp

```diff
@@ -338,7 +338,7 @@ namespace behaviour {
             emit(std::move(reset));
         }
 
-        void SoccerStrategy::penaltyShootoutLocalisationReset(const FieldDescription& fieldDescription) {
+        void SoccerStrategy::penaltyShootoutLocalisationReset() {
```

Comment:
I find it odd that we don't use this variable. It implies that we are doing something wrong and the warning is valid and should be fixed, not silenced.

Reply:
I will remove the change then?

Reply:
Either that or work out why we're not using the description of the field when we reset for a penalty.

Reply:
Do we even call this function anywhere? If we do then removing that argument would surely cause and error?

Reply:
It's called on 229

Reply:
It's called once in the same file. I removed that argument from the call.

Reply:
In that case we should put it back and figure out why we aren't using it